### PR TITLE
Minimal documentation

### PR DESCRIPTION
**Pull request by kralmichal**
Ref. https://github.com/apitte/console/issues/2

### CODE REVIEW

File: .docs/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+## Configuration
+
+```yaml
+api:
+    plugins: 
+        Apitte\Console\DI\ConsolePlugin:
+        
+services:
+	Apitte\Console\Command\RouteDumpCommand:
+		tags: [kdyby.console.command]
```
Comment (f3l1x) [inline .docs/README.md]: Well, ... Ehm.

File: .docs/README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+```
+
+## Usage
+
+Let say you would like to get list of all routes from CLI. The only thing you need to run is simple command. 
+
+`php bin/console route:dump`
```
Comment (f3l1x) [inline .docs/README.md]: It should be prefixed with the vendor name such as `apitte:route:dump` to prevent duplication.

**Review by f3l1x — CHANGES_REQUESTED** [review]

**Comment by f3l1x** [comment]
Thank you.

**Comment by kralmichal** [comment]
Thanks for merge.

**Pull request merged**